vpr/src/route/clock_connection_builders.cpp

```diff
@@ -320,7 +319,7 @@ void ClockToPinsConnection::create_switches(const ClockRRGraphBuilder& clock_gra
 
                     //Create edges depending on Fc
                     for (size_t i = 0; i < clock_network_indices.size() * fc; i++) {
-                        clock_graph.add_edge(rr_edges_to_create, clock_network_indices[i], clock_pin_node_idx, arch_switch_idx);
+                        clock_graph.add_edge(rr_edges_to_create, clock_network_indices[i], size_t(clock_pin_node_idx), arch_switch_idx);
```

Comment:
Can we change the function prototype to avoid the cast (send in an RRNodeId instead)?

Reply:
See my reply in a previous comment on the ``t_rr_edge_info``